app/server.js

```diff
@@ -1,13 +1,28 @@
-require('./insights').setup()
 const Hapi = require('@hapi/hapi')
+const config = require('./config')
+const catbox = config.useRedis ? require('@hapi/catbox-redis') : require('@hapi/catbox-memory')
+const catboxOptions = config.useRedis ? config.catboxOptions : {}
+
+require('./insights').setup()
```

Comment:
This should always be the first line executed on start up so we capture telemetry of any issues.  Should be first line in `index.js`.